File: ios/Plugin/ZoomPluginPlugin.m
#import <Foundation/Foundation.h>
#import <Capacitor/Capacitor.h>

// Define the plugin using the CAP_PLUGIN Macro, and
// each method the plugin supports using the CAP_PLUGIN_METHOD macro.
CAP_PLUGIN(ZoomPluginPlugin, "ZoomPlugin",
           CAP_PLUGIN_METHOD(enableZoom, CAPPluginReturnPromise);
           CAP_PLUGIN_METHOD(disableZoom, CAPPluginReturnPromise);
)

File: ios/Plugin/ZoomPluginPlugin.swift
import Foundation
import Capacitor

/**
 * Please read the Capacitor iOS Plugin Development Guide
 * here: https://capacitorjs.com/docs/plugins/ios
 */
@objc(ZoomPluginPlugin)
public class ZoomPluginPlugin: CAPPlugin {
    private let implementation = ZoomPlugin()

    @objc func enableZoom(_ call: CAPPluginCall) {
        call.resolve([:])
    }

    @objc func disableZoom(_ call: CAPPluginCall) {
        call.resolve([:])
    }
}
